build.sbt

```diff
@@ -115,6 +115,7 @@ dockerfile in docker := {
 
   new Dockerfile {
     from("anapsix/alpine-java:8_server-jre")
+    label("ergo", "ergo")
```

Comment:
I suppose this image is made exclusively for integration tests (we have `Dockerfile` in the root folder). I suggest we make the label more specific (e.g. "ergo-integration-tests"). Otherwise, we risk deleting wrong images since such label can be used in the future for other types of images (demo, experiments, etc.).

Reply:
it's never used, but ok, let's use `ergo-integration-tests`